Prototypes.lua

```diff
@@ -1786,39 +1786,63 @@ WeakAuras.event_prototypes = {
     name = L["DBM Announce"],
     init = function(trigger)
       WeakAuras.RegisterDBMCallback("DBM_Announce");
-      return "";
+      local ret = "local use_cloneId = %s;"
+      return ret:format(trigger.use_cloneId and "true" or "false");
     end,
+    statesParameter = "all",
+    canHaveAuto = true,
     args = {
       {
         name = "message",
         init = "arg",
         display = L["Message"],
-        type = "longstring"
-      }
+        type = "longstring",
+        store = true
+      },
+      {
+        name = "name",
```

Comment:
Hrm why not make ´store=true´ the default since that is what almost every trigger wants and set ´store=false´ for those who don't? Do we have ones that don't? If yes, why don't they want to store?